api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/pm/PasswordManagementProperties.java

```diff
@@ -266,6 +266,16 @@ public ForgotUsername() {
          */
         private boolean securityQuestionsEnabled = true;
 
+        /**
+         * Whether the Password Management Token will contain the server IP Address.
+         */
+        private boolean checkServerIpAddress = true;
```

Comment:
Rename to `includeServerIpAddress` or something similar

Reply:
Done